lib/matplotlib/widgets.py

```diff
@@ -3051,48 +3052,53 @@ def _release(self, event):
         return False
 
     def _onmove(self, event):
-        """Motion notify event handler."""
+        """
+        Motion notify event handler.
+
+        This can do one of four things:
+        - Translate
+        - Rotate
+        - Re-size
+        - Contine the creation of a new shape
+        """
+        xy = np.array([event.xdata, event.ydata])
+        xy_press = np.array([self._eventpress.xdata, self._eventpress.ydata])
 
         state = self._state
         rotate = ('rotate' in state and
                   self._active_handle in self._corner_order)
-        eventpress = self._eventpress
-        # The calculations are done for rotation at zero: we apply inverse
-        # transformation to events except when we rotate and move
         move = self._active_handle == 'C'
-        if not move and not rotate:
+        resize = self._active_handle and not move
+
+        if resize:
+            # If resizing, use xy/xy_press in a de-rotated frame
```

Comment:
```suggestion
            # use xy/xy_press in a de-rotated frame
```
Optional: The code above aleady says it.